src/theme.jsx

```diff
@@ -91,6 +91,7 @@ const theme = createMuiTheme({
       root: {
         width: 41,
         height: 41,
+        color: '#000',
```

Comment:
added this to override the default value ` "rgba(0, 0, 0, 0.54)"` in https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/IconButton/IconButton.js#L21